src/Logging/src/DynamicSerilogCore/SerilogWebHostBuilderExtensions.cs

```diff
@@ -15,7 +15,7 @@ public static class SerilogWebHostBuilderExtensions
         /// </summary>
         /// <param name="hostBuilder">The <see cref="IWebHostBuilder"/> to configure</param>
         /// <param name="configureLogger">The delegate for configuring the <see cref="DynamicLoggerConfiguration" /> that will be used to construct a <see cref="Serilog.Core.Logger" /></param>
-        /// <param name="preserveStaticLogger">Indicates whether to preserve the value of <see cref="Log.Logger"/>.</param>
+        /// <param name="preserveStaticLogger">Not Supported!</param>
```

Comment:
Although we gave this option, it never worked - cannot use static logger and dyanmically allow setting logs.

Reply:
If that's the case we should probably mark this signature obsolete and provide a more accurate option

Reply:
Fixed.